### ISSUE TITLE
Pending Quotation's Foreign Currency amount in Email Digest

### ISSUE THREAD

**Issue by tun-tun-aung** [frappe/erpnext#11267]
Pending Quotation's Foreign Currency amount show wrongly in Local Currency in email from Email Digest

**Comment by umairsy** [comment]
Please share more data to elaborate on this issue.

**Comment by tun-tun-aung** [comment]
Dear Umair,

Thanks for your reply.
Please check below screen shot.[image: Inline image 1]
Above screen shot is the quotation amount with USD499.8

[image: Inline image 3]
I got Daily Digest email with that Quotation, with same amount, different
currency.

Let me know if you need more info.

-----------------
Best Regards,
Tun Tun Aung

**Comment by tundebabzy** [comment]
@tun-tun-aung 
Please add the attachments directly on github

**Comment by tun-tun-aung** [comment]
![quotation](https://user-images.githubusercontent.com/32463398/31940688-0bf364ba-b8e5-11e7-86bd-ad4607cd6bad.jpg)
![daily digest](https://user-images.githubusercontent.com/32463398/31940709-1c0d7cf0-b8e5-11e7-8905-6f5fae1dfcf5.jpg)
![email](https://user-images.githubusercontent.com/32463398/31940717-215212ac-b8e5-11e7-97c7-b7715fb734a0.jpg)

**Comment by tun-tun-aung** [comment]
yes still.  all foreign currency amount of will display with local currency code.
